src/main/resources/apps/pantheon/modules/all/html.html

```diff
@@ -1,86 +1,9 @@
 <html>
-<head data-sly-include="${'/apps/pantheon/partials/header.html'}">
+<head>
 </head>
 <body>
-
-<div class="container-fluid" data-sly-use.moduleData="com.redhat.pantheon.use.ModuleData">
```

Comment:
I am pretty sure we can get rid of your DataRetriever class and bring the query functionality into the existing ModuleData class. To do that, change this line to something like this:
`<div class="container-fluid" data-sly-use.moduleData="${'com.redhat.pantheon.use.ModuleData' @ benTestVar='Hello world'}">`
...and then in the ModuleData class, you can use code like this to retrieve the value in the init() method:
`log.info("Ben Test Var: " + bindings.get("benTestVar"));`

Reply:
My intention here is to combine this technique with my other comment. Use data-sly-test to figure out whether the user specified a 'search' param on the URL, and depending on that, pass the correct value into the ModuleData class via this. Then we can eliminate both the search.html and the DataRetriever.java files.

Reply:
awesome I get it, I was guiding my decision on the searches I did looking on ways to pass parameters to a method thru sly which ended up with finding that I could only pass parameters down to the constructor of the class. But this is passing it via the bindings... This is way better than refactoring the code like I was doing.